bundles/org.smarthomej.binding.deconz/src/main/java/org/smarthomej/binding/deconz/internal/Util.java

```diff
@@ -106,7 +105,7 @@ public static DateTimeType convertTimestampToDateTime(String timestamp) {
      * @param value the value to find in the map
      * @return Stream of all keys for the value
      */
-    public static <@NonNull K, @NonNull V> Stream<K> getKeysFromValue(Map<K, V> map, V value) {
+    public static <K, V> Stream<K> getKeysFromValue(Map<K, V> map, V value) {
```

Comment:
Is there a reason why you changed that? We have to decide here: either a SAT warning (if we use @NonNull) or a compile warning (if we don‘t use it).

Reply:
Reverted. I removed it because of SAT warning.